server/api.js

```diff
@@ -417,4 +417,20 @@ router.post("/sign-in", async (req, res) => {
 	}
 });
 
+router.post("/logout", async (req, res) => {
+	try {
+		const authHeader = req.headers.authorization;
+		const token = authHeader?.split(" ")[1];
+
+		if (!token) {
+			return res.status(401).json({ code: "UNAUTHORIZED" });
+		}
```

Comment:
You don't need to do this, just add `/logout` to the allowlist with a value of `token`